index.html

```diff
@@ -33,6 +33,151 @@ <h3>LET’S CONNECT</h3>
           </ul>
         </div>
     </section> 
+    <section class="works" id="works">
+      <div class="cards" id="card1">
+        <div class="snapshoot-placeholders">
+          <img class="snapshoot" src="images/SnapshootD1.jpg" alt="Project-1 Snapshoot">
+        </div>
+        <div class="projects">
+          <h2 class="project-titles">Tonic</h2>
+          <ul class="inline-li project-summary">
+            <li class="client">CANOPY</li>
+            <li class="type">Back End Dev</li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="project-descriptions">
+            A daily selection of privately personalized reads; 
+            no accounts or sign-ups required.
+          </p>
+          <ul class="inline-li">
+            <li class="tags">html</li>
+            <li class="tags">css</li>
+            <li class="tags">javascripts</li>
+          </ul>
+          <button class="btn" type="button">See Project</button>
```

Comment:
- [ ] Great job adding button interactions according to requirements. You have added interaction for `hover` and `action` both accordingly. The only thing different is the font color of the button text upon the `pressed` action. The font color of your `pressed` button is `blue` whereas it should be `white` according to the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=tG1jh7hS4l814OsZ-0). Please make sure to implement it accordingly 

**_Figma design_**

<img width="353" alt="Screenshot 2023-02-17 at 11 55 26 AM" src="https://user-images.githubusercontent.com/44798044/219587783-3a8d86ea-54f2-4aa9-9120-3f685e0c42f6.png">

**_Your implementation_**

https://user-images.githubusercontent.com/44798044/219587964-31cff487-6791-4e30-b121-5fc2828870b2.mov

Reply:
Done